src/components/BirthdayPicker.tsx

```diff
@@ -0,0 +1,30 @@
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import {useState} from "react";
+import "./birthday.scss";
+
+const BirthdayPicker = () => {
+  const [value, setValue] = useState<Date | null>(null);
+  const fieldHandler = (date: Date) => {
+    setValue(date);
+  }
+  const minYear = new Date((new Date()).getTime() - 120 * 365.2 * 86400000 - 6 * 86400000 + 29.7 * 60000);
```

Comment:
too complex calculations to understand what minYear is expected to be...

What is expected to be here?

Reply:
I need Date 120 years ago, every day date should chenge

Reply:
For such puprorses it is recommended to use 'momentjs' library.

Please, leave commented description that it is 120 years ago date

Reply:
Ok, Thanks)